internal/syncer/connector/rest_api_test.go

```
@@ -0,0 +1,1185 @@
+}
+
+func TestRestAPISyncSessionPageResume(t *testing.T) {
+	withRestAPITestHooks(t)
+	var mu sync.Mutex
+	var requested []string
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		page := r.URL.Query().Get("page")
+		mu.Lock()
+		requested = append(requested, page)
+		mu.Unlock()
+		items := []any{}
```

golangci-lint (2.12.2)

[error] ineffectual assignment to items

(ineffassign)